src/main/java/edu/utdallas/davisbase/compiler/Compiler.java

```diff
@@ -252,6 +253,12 @@ public DataType convertToDavisType(Expression type)throws CompileException{
               }
               throw new CompileException("Expected an TINYINT value, but found a BIGINT value.");
 
+            case YEAR:
+	          if (Byte.MIN_VALUE <= longValue && longValue <= Byte.MAX_VALUE) {
+	            return (byte) longValue;
```

Comment:
Why is this being returned as a `byte`?

_(I know when we serialize it it should only take up 1 byte of space, but this should be represented pre-serialization using the [`java.time.Year`](https://docs.oracle.com/javase/8/docs/api/java/time/Year.html) class.)_